string scene_slug
string activity_type
string[] instances
